packages/lesswrong/lib/collections/moderationTemplates/schema.ts

```diff
@@ -0,0 +1,51 @@
+import {schemaDefaultValue} from "../../collectionUtils";
+import SimpleSchema from "simpl-schema";
+
+const ALLOWABLE_COLLECTIONS: CollectionNameString[] = ['Messages', 'Comments'];
+
+type CollectionNameString = 'Messages' | 'Comments';
```

Comment:
Redundant if defining this [here](https://github.com/ForumMagnum/ForumMagnum/pull/5983/files#diff-d41a344346869ecb15da9b661f5b600aeb1da09aea16640af8e9e6fa0e01d15aR217)